### PR TITLE
Helpful link fix

### PR DESCRIPTION
**Pull request by klAvAx**
Added target="_blank" as well as implemented the extra list thingy

### CODE REVIEW

File: webapp/src/features/home/Home.tsx [hunk 1]
```diff
@@ -11,16 +11,20 @@ const helpfulLinks = [
...
-        url: "https://docs.google.com/spreadsheets/d/1xzejwFYiaFkS7atfWf9YJe7nmZqF_UxwZBjkigoL4DI/edit?usp=sharing",
-    },
-    {
-        title: "Streaming Assets",
-        text: "Images and logos that are used in our streams, including DJ logos.",
-        url: "https://drive.google.com/drive/u/1/folders/1tuZddCNBcFpVFjyk3XKB6M1d64mAnjX8",
+        extra: [
+            {
+                title: "Signup Sheet",
+                url: "https://docs.google.com/spreadsheets/d/1ANiFOmz1_ADaMxCtcQE0HdGq3TsJM__Whg87DDN_y9Q/edit?usp=drive_link"
```
Comment (lebull) [inline webapp/src/features/home/Home.tsx:17]: Agh, the signup sheet was labeled something else in the drive.

Would you be able to swap this out with what used to be on the old "performer interest sheet" card?

https://docs.google.com/spreadsheets/d/1xzejwFYiaFkS7atfWf9YJe7nmZqF_UxwZBjkigoL4DI/edit?usp=sharing

**Review by lebull — CHANGES_REQUESTED** [review]

File: webapp/src/features/home/Home.tsx (hunk 1 above)
Comment (klAvAx) [reply webapp/src/features/home/Home.tsx:17]: Sure!

**Comment by klAvAx** [comment]
Updated the Link

**Comment by lebull** [comment]
LGTM!

**Review by lebull — APPROVED** [review]

**Pull request merged**